src/ipa_pc/data_structures.rs

```diff
@@ -105,13 +106,16 @@ impl<G: AffineCurve> PCCommitment for Commitment<G> {
     }
 
     fn size_in_bytes(&self) -> usize {
-        algebra_core::to_bytes![G::zero()].unwrap().len() / 2
+        to_bytes![G::zero()].unwrap().len() / 2
     }
 }
 
 impl<G: AffineCurve> ToBytes for Commitment<G> {
```

Comment:
Here, you could `#[derive(CanonicalSerialize, CanonicalDeserialize)]` on the struct instead of having to manually impl it.

Same applies for all follow-up impls of the `size_in_bytes` and the `write` functions inside their respective traits